api/s2n.h

```diff
@@ -595,25 +2079,118 @@ extern int s2n_session_ticket_get_data(struct s2n_session_ticket *ticket, size_t
 S2N_API
 extern int s2n_session_ticket_get_lifetime(struct s2n_session_ticket *ticket, uint32_t *session_lifetime);
 
+/**
+ * De-serializes the session state and updates the connection accordingly.
+ *
+ * @note This is for < TLS 1.2 session resumption.
```

Comment:
This is incorrect. This fucntion is also used for TLS1.3.

Reply:
Does this comment apply to all the session APIs with this note?

Reply:
The only function that doesn't work well for TLS1.3 is s2n_connection_get_session(), because it only returns one ticket when in TLS1.3 you can send many tickets. So you only need a note for s2n_connection_get_session.

Reply:
Ah great! Thanks!